src/service/member-api-adapter/index.js

```diff
@@ -41,9 +41,19 @@ export default function MemberAPIService(config) {
         return serviceOptions;
     };
 
+    const generateUserQuery = function (params) {
+        if (params.hasOwnProperty('userIds')) {
```

Comment:
Given that most of the methods already take in a string `userId` I think adding another property `userIds` will be confusing -- instead have `userId` take in `string` or `string[]` instead and update jsdoc to match.

Reply:
Also update one of the test-cases in `test-member-api-service` to account for array userids